### DISCUSSION TITLE
Custom Form Validation UI

### DISCUSSION THREAD

**Discussion by webup98** [rsuite/rsuite#1546] [General]
It's really bad when the developer  can't control the UI of the error messages.
I suggest to separate this in a prop called errors for ex. and let users display them as wanted. 
Thanks in advance!

**Comment by simonguo** [comment]
![image](https://user-images.githubusercontent.com/1203827/110445296-ec526680-80f8-11eb-8683-f4f03c7ad973.png)
https://github.com/rsuite/rsuite/blob/master/docs/pages/components/form/en-US/error-message.md
Hope this example can help you.
